src/controllers/TorusController.ts

```diff
@@ -1577,7 +1450,6 @@ export default class TorusController extends BaseController<TorusControllerConfi
   private async requestAccounts(req: JRPCRequest<unknown>): Promise<string[]> {
     // Try to restore from backend (restore privatekey)
     this.embedController.update({ loginInProgress: true });
-    await this.restoreFromBackend();
     this.embedController.update({ loginInProgress: false });
```

Comment:
this needs to change...

Reply:
the flow needs to wait for openlogin session to resolve and then, respond here

Reply:
Added to resolve openlogin